src/components/resources-section/resources-section/ResourcesSection.js

```diff
@@ -59,7 +67,10 @@ const ResourcesSection = ({ learningUnitId }) => {
       <Panel header={header}>
         <LearningUnitInformation learningUnit={learningUnit} />
         {displayBasic && <AddNewResourceModal handlers={modalHandlers} learningUnitId={learningUnitId} />}
-        <ResourcesList resources={resources} learningUnitId={learningUnitId} />
+        <ResourcesList resources={resources} learningUnitId={learningUnitId} resourceViewButtonHandler={(resource) => resourceViewButtonHandler(resource)} />
```

Comment:
I think you have to pass here a "newEvaluationSubmited" that handles changes when the user adds a Resource Evaluation, since it is not showing on the sidebar when I submit an Evaluation, escape and press the same Resource again.

e.g: 
1) Without Evaluation
![image](https://user-images.githubusercontent.com/12021066/197585570-abf2a6eb-9e7b-493d-b9cd-c6727bcf6715.png)

2) Submitted 
![image](https://user-images.githubusercontent.com/12021066/197585625-e4f47d88-c149-410b-a822-0a67df2ec1c0.png)

3) Submitted but not showing when return to resource
![image](https://user-images.githubusercontent.com/12021066/197585765-9c3521ca-0caa-4e44-83e8-ceb3151355b4.png)

`note: Third photo is not referring to the same resource, but it makes the point`

Reply:
I finally solved this issue. We needed to call the mutate method of resourceEvaluation endpoint. It was not enough calling the mutate on resourceEvaluations.

Reply:
Bien!